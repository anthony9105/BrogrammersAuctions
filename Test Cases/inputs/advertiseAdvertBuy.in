login 
newUser1234 
advertise 
logout 