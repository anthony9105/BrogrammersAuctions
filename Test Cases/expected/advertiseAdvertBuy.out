Welcome to the Auction System
Please login to begin. 
Welcome, newUser1234 
Cannot advertise as standard-buy 
Session complete.
